unicodetools/src/main/java/org/unicode/idna/GenerateIdnaTest.java

```diff
@@ -169,7 +169,7 @@ int generateTests(int lines) throws IOException {
             // See https://github.com/unicode-org/unicodetools/issues/100 "Whither 13.1.0?"
             lastVersion = "13.0.0";
         }
-        int ucdTypesLastVersion = UCD_Types.AGE130;
+        int ucdTypesLastVersion = UCD_Types.AGE140;
```

Comment:
FYI  The race is on between this PR and PR #234 :-)

Reply:
Aww, runner up. I updated from latest on `main`, so this diff disappeared from the PR.